internal/xds/rbac/rbac_engine_test.go

```diff
@@ -424,16 +435,182 @@ func (s) TestNewChainEngine(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "SimpleAuditLogger",
+			policies: []*v3rbacpb.RBAC{
+				{
+					Action: v3rbacpb.RBAC_ALLOW,
+					Policies: map[string]*v3rbacpb.Policy{
+						"anyone": {
+							Permissions: []*v3rbacpb.Permission{
+								{Rule: &v3rbacpb.Permission_Any{Any: true}},
+							},
+							Principals: []*v3rbacpb.Principal{
+								{Identifier: &v3rbacpb.Principal_Any{Any: true}},
+							},
+						},
+					},
+					AuditLoggingOptions: &v3rbacpb.RBAC_AuditLoggingOptions{
```

Comment:
I see that you're using AuditLoggingOptions part of RBAC, which isn't available for
github.com/envoyproxy/go-control-plane v0.11.0 dependency
Did you update the version of the dependency in go.mod but didn't commit it? Or am I missing something?

Reply:
It got updated in the previous PR https://github.com/grpc/grpc-go/pull/6192